### PR TITLE
13.0. stock_warehouse_orderpoint_stock_info: New filters under minumum and …

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
Add two new filters in stock warehouse stock, products over maximum and under minimum.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@manuelcalerosolis Could you add few tests for searches ?

**Comment by manuelcalerosolis** [comment]
> Could you add few tests for searches ?

@rousseldenis Can you tell me where I can find tests for the filters in other oca modules? To inspire me, I have never done them.

File: stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py [hunk 1]
```diff
@@ -29,30 +29,60 @@ class StockWarehouseOrderpoint(models.Model):
...
+                        "product_under_minimum": product["qty_available"]
+                        < order.product_min_qty,
+                        "product_over_maximum": product["qty_available"]
+                        > order.product_max_qty,
                     }
                 )
+
+    def _search_product_under_minimum(self, operator, value):
+        if operator == "=":
+            recs = self.search([]).filtered(lambda x: x.product_under_minimum is True)
```
Comment (rousseldenis) [inline stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py:76]: @manuelcalerosolis Write a test that searches for product with the field ('product_under_minimum', '=', True).

Moreover, IMHO this is a little bulky as it result of two searches (one with all products, then a search with ids).

Comment (manuelcalerosolis) [reply stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py:76]: @rousseldenis certainly the search is very bad, currently the module is not being used in version 13.0 I will prepare it for version 15.0

**Pull request closed without merging**